worldedit-core/src/main/java/com/sk89q/worldedit/session/PasteBuilder.java

```diff
@@ -101,6 +104,21 @@ public PasteBuilder ignoreAirBlocks(boolean ignoreAirBlocks) {
         return this;
     }
 
+    /**
+     * Set whether structure void blocks in the source are skipped over when pasting.
+     *
+     * <p>
+     * This defaults to true, to better align to how Minecraft intends these blocks to function.
```

Comment:
docs should be updated to a) mention the correct default and b) mention the intent to change in a major-version update.
it's best if people explicitly set this to whatever they want to avoid defaulting to something they might not.